wp-content/plugins/cf-demo-lessons/cf-demo-lessons.php

```diff
@@ -114,17 +114,17 @@ public function registerLessonMetabox()
 
 			$cmb->add_field(
 				array(
-					'name' => esc_html__('Promoted', 'cmb2'),
-					'desc' => esc_html__('Promoted field', 'cmb2'),
+					'name' => esc_html__('Promoted', 'cf-demo-lessons'),
+					'desc' => esc_html__('Promoted field', 'cf-demo-lessons'),
 					'id' => 'promoted_checkbox',
 					'type' => 'checkbox',
 				)
 			);
 
 			$cmb->add_field(
 				array(
-					'name' => __('Order', 'cmb2'),
-					'desc' => __('field Order Number', 'cmb2'),
+					'name' => __('Order', 'cf-demo-lessons'),
+					'desc' => __('field Order Number', 'cf-demo-lessons'),
```

Comment:
The description field should give a little insight about what that field *does*. "field Order Number" does not provide additional information to the field title "Order". A better description would be "This is used to order query results". Always start with capital letter. If no description is needed (the field usage is obvious), leave it empty.